test/middleware/fuse_test.exs

```diff
@@ -39,7 +39,7 @@ defmodule Hardhat.FuseTest do
       Bypass.down(bypass)
 
       for _ <- 0..3 do
-        assert {:error, "connection refused"} = TestClient.get("http://localhost:#{bypass.port}/")
+        assert {:error, :econnrefused} = TestClient.get("http://localhost:#{bypass.port}/")
```

Comment:
I think (but am not 100% sure) that this is needed due to https://github.com/elixir-tesla/tesla/compare/v1.8.0...v1.9.0#diff-3259633d9aaf85fa9e230ca60bf0afb4ab3f6e0b44d4d61f4e72ad4101399496L63-L70.